script.js

```diff
@@ -30,3 +30,104 @@ function menuDisappear() {
+        projectDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+        projTechno: ['Ruby on Rails', 'CSS', 'JavaScript'],
+        linkToLive: 'https://nahid1911.github.io',
+        linkToSource: 'https://github.com/Nahid1911/Nahid-s_Portfolio',
+      },
+
+      {
+        projectName: 'Keeping track of hundreds of components',
+        projectDescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.',
+        projTechno: ['Ruby on Rails', 'CSS', 'JavaScript'],
+        linkToLive: 'https://nahid1911.github.io',
+        linkToSource: 'https://github.com/Nahid1911/Nahid-s_Portfolio',
+      },
+    ];
+
+    const div1 = document.createElement('div');
+    div1.id = 'mainContainerId';
+    div1.className = 'mainContainer';
+    div1.innerHTML = `
+      <div id="mainContainerL2" class="mainContainerL2">
+        <div id="children0" class="children0">
+          <img id="children0Img" src="media/popup_Icon_mobile_Cancel.svg">
+        </div><div id="children1" class="children1"></div>
+        <div id="children2" class="children2">
+          <h2>${projects[i].projectName}</h2>
+          <ul id="ulChildren2Class" class="ulChildren2Class">
+            <li id="liUlChildren2Class" class="liUlChildren2Class">${projects[i].projTechno[0]}</li>
+            <li id="liUlChildren2Class" class="liUlChildren2Class">${projects[i].projTechno[1]}</li>
+            <li id="liUlChildren2Class" class="liUlChildren2Class">${projects[i].projTechno[2]}</li>
+          </ul>
+        </div>
+        <div id="children3" class="children3">
+          <p id="pOfChild3" class="pOfChild3">${projects[i].projectDescription}</p>
+        </div>
+        <div id="children4" class="children4">
+          <a href=${projects[i].linkToLive} target="_blank">
+            <button id="btn-all-id" class="btn-all">See live
+              <img src="media/Popup_img_Mobile_btn.svg">
+            </button>
+          </a>
+          <a href=${projects[i].linkToSource} target="_blank">
+            <button id="btn-all-id2" class="btn-all">See source
+              <img src="media/Popup_img_mobile_btn_GitHub.svg">
+            </button>  
+          </a>
+        </div>
+      </div>`;
+    document.body.appendChild(div1);
+
+    // Mobile-Menu-Disapper
+
+    const toggleBtn = document.getElementById('children0Img');
+    const toggleContent = document.getElementById('mainContainerId');
+    toggleBtn.addEventListener('click', () => {
+      if (toggleContent) {
+        toggleContent.parentNode.removeChild(toggleContent);
+      }
+    });
+    div1.scrollIntoView({ behavior: 'smooth', block: 'start' });
```

Comment:
[OPTIONAL] It is impressive how closely you implement the design without judging. 👨🏽‍⚖️ Yes, you are right, when pop-up opens, it is at the top of the page according to the Figma template. But I highly recommend you to fix it, so pop up stays in the position where it was opened. Otherwise, when opening a popup for each project we need to go back down to the projects section and this is a painful UX. 😭 You can check this [project](https://huseyinbiyik.github.io/) to understand what I mean with the popup behaviour.

Reply:
Have resolved this issue also.